src/composables/queries/usePoolQuery.ts

```diff
@@ -91,8 +94,12 @@ export default function usePoolQuery(
       pool.wrappedTokens[index] = getAddress(
         linearPool.tokensList[linearPool.wrappedIndex]
       );
+
+      Object.assign(tokensMap, keyBy(linearPool.tokens, 'address'));
     });
 
+    pool.underlyingTokens = tokensMap;
```

Comment:
Not a massive fan of this wording, what about `linearPoolTokens` or `linearPoolTokenMap`

Reply:
Agreed - switched to an array (filtered without the pool token). Much cleaner.